src/test/java/org/openstreetmap/atlas/checks/validation/points/AddressPointMatchCheckTestRule.java

```diff
@@ -0,0 +1,98 @@
+package org.openstreetmap.atlas.checks.validation.points;
+
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.utilities.testing.CoreTestRule;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Edge;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Loc;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Node;
+import org.openstreetmap.atlas.utilities.testing.TestAtlas.Point;
+
+/**
+ * {@link AddressPointMatchCheckTest} data generator
+ *
+ * @author savannahostrowski
+ */
+
+public class AddressPointMatchCheckTestRule extends CoreTestRule
```

Comment:
I'd suggest writing tests with empty street name tag value for the initial point.